src/transformers/models/longformer/modeling_tf_longformer.py

```diff
@@ -1766,24 +1764,26 @@ def _pad_to_window_size(
                 )
             )
 
-            paddings = tf.constant([[0, 0], [0, padding_len]])
+        paddings = tf.convert_to_tensor([[0, 0], [0, padding_len]])
 
-            if input_ids is not None:
-                input_ids = tf.pad(input_ids, paddings, constant_values=pad_token_id)
+        if input_ids is not None:
+            input_ids = tf.pad(input_ids, paddings, constant_values=pad_token_id)
```

Comment:
When `padding_len==0`, then this won't change the `input_ids` correct?

Reply:
Correct!